result/yoda.js

```diff
@@ -0,0 +1,13 @@
+
+/*eslint no-multi-spaces: "error"*/
+
+var a =  1;
+
+if(foo   === "bar") {}
+
+a <<  b
```

Comment:
__[development][eslint]__
* Error - Multiple spaces found before 'b'. (no-multi-spaces)
* Error - 'b' is not defined. (no-undef)